util/raclgen.py

```diff
@@ -64,6 +64,7 @@ def main():
 <% group_suffix = f"_{racl_group.upper()}" if racl_group and racl_group != "Null" else "" %>\
 <% if_suffix = f"_{if_name.upper()}" if if_name else "" %>\
 <% reg_name_len = max( (len(name) for name in register_mapping.keys()) ) %>\
+<% window_name_len = 0 if len(window_mapping) == 0 else max( (len(name) for name in window_mapping.keys()) ) %>\
```

Comment:
I think this can be slightly simpler:
```python3
>>> d = {}
>>> max((len(x) for x in d.keys()), default=0)
0
```
(and similarly for `toplevel_racl_pkg.sv.tpl`)